### PR TITLE
site: site style optimization

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | site: site style optimization |
| 🇨🇳 Chinese | site: site style optimization |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: .dumi/theme/slots/Header/index.tsx [hunk 1]
```diff
@@ -130,16 +126,19 @@ const useStyle = createStyles(({ token, css }) => {
       user-select: none;
     `,
     link: css`
-      margin-left: 10px;
-
+      margin-inline-start: 100px;
```
Comment (MadCcc) [inline .dumi/theme/slots/Header/index.tsx:129]:
```suggestion
      margin-inline-start: 10px;
```
笔误？

Comment (li-jia-nan) [reply .dumi/theme/slots/Header/index.tsx:129]: 啊，写错了

File: .dumi/theme/slots/Header/index.tsx [hunk 2]
```diff
@@ -299,8 +298,8 @@ const Header: React.FC = () => {
     navigationNode,
     <Select
       key="version"
-      className="version"
```
Comment (MadCcc) [inline .dumi/theme/slots/Header/index.tsx:302]: 这个应该要去 global style 里把对应的删掉。
有不少样式还是保留在 global style 里，可以迁到 cssinjs 的

Comment (li-jia-nan) [reply .dumi/theme/slots/Header/index.tsx:302]: 我没搜到这个……以为是多余的

Comment (li-jia-nan) [reply .dumi/theme/slots/Header/index.tsx:302]: 是说这些吗？

https://github.com/ant-design/ant-design/blob/e5849fc1b9db68fa95f674110592480c33608821/.dumi/theme/common/styles/index.ts#L1-L13

Comment (li-jia-nan) [reply .dumi/theme/slots/Header/index.tsx:302]: 看了一下还蛮多的，我单开个 pr 做吧

File: .dumi/theme/slots/Header/index.tsx [hunk 3]
```diff
@@ -130,16 +126,19 @@ const useStyle = createStyles(({ token, css }) => {
...
-      width: 22px;
-      height: 22px;
+    versionSelect: css`
+      min-width: 90px;
+      .rc-virtual-list {
+        .rc-virtual-list-holder {
+          scrollbar-width: thin;
+          scrollbar-color: unset;
+        }
+      }
```
Comment (afc163) [inline .dumi/theme/slots/Header/index.tsx:141]: 直接用 ConfigProvider 搞定？

Comment (li-jia-nan) [reply .dumi/theme/slots/Header/index.tsx:141]: 这个元素没有在 CP 透出，只能用 css 写，另外不是所有的地方都是 antd 组件，也有的地方是原生的标签，用 CP 反而更麻烦了

Comment (li-jia-nan) [reply .dumi/theme/slots/Header/index.tsx:141]: 那我在 feature 分支加个 token ？

Comment (li-jia-nan) [reply .dumi/theme/slots/Header/index.tsx:141]: - #48109

**Comment by li-jia-nan** [comment]
这个先合了吧，token 要在 feature 分支上做，等那边做好了我再新开一个 pr 把相关的值替换成 token

**Review by afc163 — APPROVED** [review]

**Pull request merged**